library/src/helpers/parser.ts

```diff
@@ -59,7 +63,9 @@ class Parser {
 
   private async validate(json: JSON, schema: string): Promise<JSON> {
     return new Promise<JSON>((resolve, reject) => {
-      this.validator.validate(json, schema, (err) => {
+      this.validator.validate(json, schema, err => {
+        if (Array.isArray(err) && err.length === 1)
+          return reject(err[0].message || err[0]);
```

Comment:
For longer than `return {variable}` statement, please use `{}` after `if` statement